### ISSUE TITLE
GraphQL Aggregate `groupBy` filter gives inconsistent result

### ISSUE THREAD

**Issue by laura-ham** [weaviate/weaviate-javascript-client#8]
Important to note is that the property `isAccessible` is not filled, so it only has `null` values.

The following query in plain GraphQL gives the following result:

GraphQL Query:
```graphql
{
  Aggregate {
    Things {
      Article(groupBy: ["isAccessible"]) {
        meta {
          count
        }
        wordCount {
          mean
        }
        groupedBy {
          value
          path
        }
      }
    }
  }
}

```

GraphQL Result:
```json
{
  "data": {
    "Aggregate": {
      "Things": {
        "Article": []
      }
    }
  },
  "errors": null
}
```

Expected is the same output with the JS code, but the result is inconsistent:

JS Query:
```js
const weaviate = require("weaviate-client");

const client = weaviate.client({
  scheme: 'http',
  host: 'localhost:8080',
});

client.graphql
      .aggregate()
      .withClassName('Article')
      .withGroupBy(["isAccessible"])
      .withFields('meta { count } wordCount {mean} groupedBy {value path}')
      .do()
      .then(res => {
        console.log(JSON.stringify(res))
      })
      .catch(err => {
        console.error(err)
      });
```

JS response:
```json
{
  "data": {
    "Aggregate": {
      "Things": {
        "Article": [
          {
            "groupedBy": null,
            "meta": {
              "count": 2548
            },
            "wordCount": {
              "mean": 1013.82025
            }
          }
        ]
      }
    }
  },
  "errors": null
}
```

**Comment by etiennedi** [comment]
Should be fixed/implemented in v1.0.0-rc4. Keeping this issues open until I have feedback that it works as desired.

**Comment by etiennedi** [comment]
Not fixed, investigating with same test set now.
